### PR TITLE
Feature/36 header style

### LINKED ISSUE #36: Angular theme should look nicer for table headers
#### Describe the desired behavior

The table header row should look more like this. 
- Color: primary text
- font: subtitle-2, as [defined on doc-it](https://pxblue.github.io/style/typography#type-scale)

column-order-1

#### Describe the current behavior

image

### PR DESCRIPTION
**Pull request by emclaug2**
Fixes #36 

#### Changes proposed in this Pull Request:
- Change mat table header styles. 

#### Screenshots:
image

#### To Test:
- Run the showcase and go here: http://localhost:4200/material-components/data-display-components
- View Table

### CODE REVIEW

**Review by huayunh — COMMENTED** [review]
in dark theme, it appears to be using black200 (secondaryText) instead of black 50 (primaryText)

File: _blueTheme.scss [hunk 1]
```diff
@@ -483,4 +483,17 @@
...
+        color: map-get($foreground, text);
+    }
+    /* .mat-sort-header-arrow[style] {
+        opacity: 1!important;;
+        color: rgba(map-get($foreground, secondary-text), 1);
+    }
+    .mat-sort-header-sorted .mat-sort-header-arrow {
+        color: map-get($foreground, secondary-text);
+    }
+    */
```
Comment (huayunh) [inline _blueTheme.scss:498]: what are these comments for?

Comment (emclaug2) [reply _blueTheme.scss:498]: Ah I should have taken these out; these start to match the Sortation icon styles that Suraj is working on.

Comment (emclaug2) [reply _blueTheme.scss:498]: I'll remove them & update the dark theme color.

Comment (emclaug2) [reply _blueTheme.scss]: Updated.

**Review by huayunh — APPROVED** [review]
Very well

**Pull request merged**